packages/account/src/Constants/routes-config.ts

```diff
@@ -20,13 +20,15 @@ import {
     AccountClosed,
     DeactivateAccount,
     LanguageSettings,
-} from 'Sections';
+} from '../Sections';
+
+import { TRoute, TRouteConfig } from 'Types';
 
 // Error Routes
 const Page404 = React.lazy(() => moduleLoader(() => import(/* webpackChunkName: "404" */ 'Modules/Page404')));
 
 // Order matters
-const initRoutesConfig = ({ is_appstore }) => [
+const initRoutesConfig = (is_appstore: boolean): TRouteConfig[] => [
```

Comment:
will this work correctly at places `initRoutesConfig` is invoked? here its now a boolean parameter, but i think an object will be passed as arguments to this function. right?